soccer/gameplay/situational_play_selection.py

```diff
@@ -0,0 +1,742 @@
+import main
+import time
+import robocup
+import math
+from enum import Enum
+import constants
+from typing import List, Dict
```

Comment:
"Don't use from xxxx import yyyy where yyyy is not a module, use import xxxx. This has to do with how python handles modules and the classes loaded from those modules. If you do it the wrong way, when a play file changes on disk and we reload it as a module, the yyyy that you imported from the module earlier won't get reloaded."

See https://github.com/RoboJackets/robocup-software/blob/63d982c288342243528836695e1bb5cec56a9745/soccer/gameplay2/readme.md.

Reply:
Good to know.